ext/fg/js/float.js

```diff
@@ -162,6 +162,33 @@ class DisplayFloat extends Display {
     setContentScale(scale) {
         document.body.style.fontSize = `${scale}em`;
     }
+
+    async getDocumentTitle() {
+        try {
+            const uniqueId = yomichan.generateId(16);
+
+            const promise = yomichan.getTemporaryListenerResult(
+                chrome.runtime.onMessage,
+                ({action, params}, {resolve}) => {
+                    if (
+                        action === 'documentInformationBroadcast' &&
+                        isObject(params) &&
+                        params.uniqueId === uniqueId &&
+                        params.frameId === 0
```

Comment:
Not sure if we should rely on `frameId` being 0 for the root frame unless it's in the spec. There's already enough context from Yomichan like in
https://github.com/FooSoft/yomichan/blob/3df78904cf734da208c6fd1b6ae1cd6612323148/ext/fg/js/frontend.js#L261-L265
(that `frameId` is used for PopupProxy parent frame ID)

Reply:
Checking popup depth 0 on search page popup might be a problem, though, because it's started from 1 on purpose to account for the page itself.

Reply:
> Not sure if we should rely on frameId being 0 for the root frame unless it's in the spec.

https://developer.chrome.com/extensions/webNavigation#frame_ids

>  Frames within a tab can be identified by a frame ID. The frame ID of the main frame is always 0, the ID of child frames is a positive number. Once a document is constructed in a frame, its frame ID remains constant during the lifetime of the document. As of Chrome 49, this ID is also constant for the lifetime of the frame (across multiple navigations).

Reply:
Additionally, `if (!this.popup.isProxy() && this.popup.depth === 0) {` can have multiple matches when iframes are involved.

Reply:
Right, it seems that the check is only valid when iframe popups are shown in root frame.

I got the impression that frame id 0 for root frame was an implementation detail from an earlier comment somewhere, but it seems quite reliable based on the Chrome document.